modules/retail/pages/retail-terminal-names.adoc

```diff
@@ -14,5 +30,115 @@ For example:
 B002.TOSHIBA-6140100-41BA03X-c643
 ----
 
-You can adjust this behavior by toggling the [systemitem]``DISABLE_HOSTNAME_ID`` and [systemitem]``DISABLE_UNIQUE_SUFFIX`` parameters in the PXE formula settings.
-For more information about the PXE formula, see xref:retail:retail-formulas.adoc[].
+
+The branch ID is the unique identifier for the branch server that the terminal is connected to.
+You can configure this value in the PXE formula settings for the branch server.
+
+The manufacturer, product name, and serial number are provided by the terminal hardware BIOS.
+If the terminal does not provide a serial number, it will be replaced in the terminal name with a hyphen (``-``).
```

Comment:
This is not entirely correct. It the terminal hardware does not provide serial number, then serial number will be omitted.